### PR TITLE
feat(chat-e2e): implemented tests for prompt folders enumeration

### PR DESCRIPTION
**Pull request by nartovm**
**Description:**

New tests for EPMRTC-1619, EPMRTC-1620, EPMRTC-1566, EPMRTC-2983, EPMRTC-2984, EPMRTC-2986, EPMRTC-2987, EPMRTC-2988, EPMRTC-2989, EPMRTC-2990

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [ ] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by nartovm** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1629.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat-e2e/src/ui/selectors/dialogSelectors.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export const PromptModal = {
   promptDescription: '[data-qa="prompt-descr"]',
   promptValue: '[data-qa="prompt-value"]',
   savePrompt: '[data-qa="save-prompt"]',
+  closePrompt: 'svg.tabler-icon.tabler-icon-x',
```
Comment (irinakartun) [inline apps/chat-e2e/src/ui/selectors/dialogSelectors.ts]: 'close' icon selector has already been defined in `IconSelectors.cancelIcon`

File: apps/chat-e2e/src/ui/selectors/sideBarSelectors.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ export const PromptBarSelectors = {
   newFolder: '[data-qa="create-prompt-folder"]',
   promptFolders: '[data-qa="prompt-folders"]',
   newPromptButton: '[data-qa="new-prompt"]',
-  prompts: '[data-qa="prompts"]',
+  prompts: '[data-qa="promps-section-container"] >> [data-qa="prompts"]',
```
Comment (irinakartun) [inline apps/chat-e2e/src/ui/selectors/sideBarSelectors.ts]: could you please fix typo 'promps' in `apps/chat/src/components/Promptbar/components/Prompts.tsx`

Comment (nartovm) [reply apps/chat-e2e/src/ui/selectors/sideBarSelectors.ts:40]: It will not work, it should be changed on the UI-side

Comment (nartovm) [reply apps/chat-e2e/src/ui/selectors/sideBarSelectors.ts:40]: image

File: apps/chat-e2e/src/tests/promptNumeration.test.ts [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+          folderPrompts.getFolderByName(
+            ExpectedConstants.newPromptFolderWithIndexTitle(2),
+          ),
+          {
+            isHttpMethodTriggered: false,
+          },
+        );
+        await promptBar.dragAndDropEntityToFolder(
+          folderPrompts.getFolderEntity('New folder 1', duplicatedPromptName),
+          folderPrompts.getFolderByName('New folder 2'),
```
Comment (irinakartun) [inline apps/chat-e2e/src/tests/promptNumeration.test.ts]: please, replace 'New folder 1', 'New folder 2' with corresponding const

File: apps/chat-e2e/src/ui/selectors/dialogSelectors.ts [hunk 4]
```diff
@@ -17,6 +17,7 @@ export const PromptModal = {
   promptDescription: '[data-qa="prompt-descr"]',
   promptValue: '[data-qa="prompt-value"]',
   savePrompt: '[data-qa="save-prompt"]',
+  // closePrompt: 'svg.tabler-icon.tabler-icon-x',
```
Comment (irinakartun) [inline apps/chat-e2e/src/ui/selectors/dialogSelectors.ts]: please, remove not used selector

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1629.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by irinakartun** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1629.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat-e2e/src/ui/selectors/dialogSelectors.ts (hunk 1 above)
Comment (nartovm) [reply apps/chat-e2e/src/ui/selectors/dialogSelectors.ts]: fixed

File: apps/chat-e2e/src/ui/selectors/sideBarSelectors.ts (hunk 2 above)
Comment (nartovm) [reply apps/chat-e2e/src/ui/selectors/sideBarSelectors.ts]: fixed

File: apps/chat-e2e/src/tests/promptNumeration.test.ts (hunk 3 above)
Comment (nartovm) [reply apps/chat-e2e/src/tests/promptNumeration.test.ts]: Fixed

File: apps/chat-e2e/src/ui/selectors/dialogSelectors.ts (hunk 4 above)
Comment (nartovm) [reply apps/chat-e2e/src/ui/selectors/dialogSelectors.ts]: fixed

**Comment by nartovm** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1629.nightly-test.deltixhub.io
>E2E tests status: success

**Review by irinakartun — APPROVED** [review]

**Comment by nartovm** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1629.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**